docs/UserGuide.md

```diff
@@ -2,13 +2,10 @@
 layout: page
 title: User Guide
 ---
-Have you ever been overwhelmed by the number of assignments given by your module instructors? How amazing would it be 
-if you just have a helper, to note down those things for you, and remind you as the deadline gets closer. If 
-this is you, then ModuleBook3.5 is the right fit for you.
-
 ModuleBook3.5 is the go-to tool for busy students/professionals who are confident that typing can save them time. 
 Using ModuleBook3.5, one can organise and keep track of tasks and their deadlines without the need for consistent 
-internet connection.
+internet connection. Even if your online learning portal fails, you can still see your task details
```

Comment:
should we mention learning portal here? cuz that is like a subset of what moduleBook3.5 sets out to do?

Reply:
I chose to mention learning portal here since some features of MB-3.5 can also be found in LumiNUS (like showing which tasks have closer deadlines). Not sure what else to add though.